mrp_workcenter_hierarchical/README.rst

```diff
@@ -0,0 +1,17 @@
+Description
+===========
+
+See __openerp__.py
```

Comment:
This is not correct. The __openerp__ should be put here and remove it from the manifest.